apps/playground/src/app/modules/components/footer-module/footer-module.component.ts

```diff
@@ -273,7 +264,13 @@ const partners: FooterPartner[] = [
   styleUrls: ['./footer-module.component.scss']
 })
 export class FooterModuleComponent implements OnInit, OnDestroy {
-  formFooter: FormGroup<FormFooter> | undefined;
+  formFooter: FormGroup =  this.formBuilder.group({
```

Comment:
Please remove the typing otherwise it won't work as assigning to `FormGroup` means assigning by default to `FormGroup<any>`. Therefore, you're overriding all the typing information that TypeScript could infer from the value.